src/appShell/App/PortalFooter.tsx

```diff
@@ -33,11 +35,15 @@ export default class PortalFooter extends React.Component<{}, IPortalFooterState
     }
 
     render() {
+            var version = "--";
```

Comment:
One more thing: if you look at PortalHeader and Container.tsx, you'll see how  appStore is passed in as a property.  This is better than having global references everywhere so lets adopt that here too